README.md

```diff
@@ -141,12 +141,22 @@ sudo apt-get install \
 
 In the folly directory, run:
 ```
-  mkdir _build && cd _build
+  mkdir _build
```

Comment:
The line below is `cmake ..`; that makes this change incorrect.

The reason for building from within the `_build` dir is to avoid cmake polluting the top-level dir with build artifacts, which it otherwise would do.

Reply:
Hmm do you mean the top level directory that contains the `.md` files and the `build`, `folly` and `CMake` folders? No build artifacts were added there. It's within the `folly` folder that a few new files and folders were created. More so, the commands just didn't work inside `[folly version]/folly/_build`. #847 contains the error I got from running `cmake ..` inside it, but both `make -j  $(nproc)` and `make install` gave errors that indicated necessary files were missing, and then they worked when I went back from `/[folly version]/folly/_build/` to `[folly version]/folly/` as @Orvid suggested.

Reply:
Right, top-level or `folly/` subdir - we want to avoid build artifact pollution.

Was your shell in `[folly-checkout]/folly/_build` or in `[folly-checkout]/_build` when you ran `cmake ..`?

Reply:
It was in `[folly-checkout]/folly/_build` when I ran it and got the errors. Should the `_build` folder have been made in `[folly-checkout]/` rather than `[folly-checkout]/folly/`? The README says
> In the folly directory

so I interpreted that as `[folly-checkout]/folly`.

Reply:
The shell should be in `[folly-checkout]/_build`, and `cmake ..` should be run from that directory.

Reply:
Okay, thanks for clarifying. So what do you think of these changes:

In the root of the folly directory, run:
```
mkdir _build && cd _build # puts the shell in [folly-checkout]/_build
```

Reply:
I don't see that as a helpful comment because it merely restates the code into English.

Reply:
"The X directory" and "the root of the X directory" mean the same thing. However you might say

> In the folly directory (e.g. the checkout root or the archive unpack root), run:

Reply:
Okay, thank you. I have updated the code.